modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSViews.tsx

```diff
@@ -31,6 +31,7 @@ const DEFAULT_ITEMS_PER_PAGE = 20;
 type FDSViewType = {
 	[OBJECT_RELATIONSHIP.FDS_ENTRY_FDS_VIEW]: FDSEntryType;
 	defaultItemsPerPage: number;
+	defaultView: string;
```

Comment:
if you change the persistence field for `defaultVisualizationMode`, this one has to be changed as well